src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Circle implements Figure {
+    private int radius;
+
+    public Circle(String color, int radius) {
```

Comment:
The constructor should also initialize a color field, but it seems like the color parameter is not being used. Make sure to add a color field to the class and initialize it in the constructor.